### PR TITLE
JAS-971 | Share the user demo on confirm

### PR DESCRIPTION
**Pull request by avghadsangita**

### CODE REVIEW

File: src/In.ProjectEKA.HipService/Link/Model/Auth.cs [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 {
-    public class AuthInit
+    public class Auth
     {
-        public AuthInit(string transactionId, AuthType authType, Meta meta)
+        public Auth(string transactionId, Meta meta,Mode mode)
         {
             TransactionId = transactionId;
-            AuthType = authType;
+         //   AuthType = authType;
```
Comment (ganesanarun) [inline src/In.ProjectEKA.HipService/Link/Model/Auth.cs:8]: Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6173534)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/Link/Model/Auth.cs  1
- src/In.ProjectEKA.HipService/Link/Model/Patient.cs  1
- src/In.ProjectEKA.HipService/User/UserController.cs  3
- src/In.ProjectEKA.HipService/Link/Model/Address.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6173534&bid=20154212)

File: src/In.ProjectEKA.HipService/Link/Model/Auth.cs [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
             Meta = meta;
+            Mode = mode;
         }
 
         public string TransactionId { get; }
-        public AuthType AuthType { get; }
+       // public AuthType AuthType { get; }
```
Comment (ganesanarun) [inline src/In.ProjectEKA.HipService/Link/Model/Auth.cs:14]: Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6173534)

File: src/In.ProjectEKA.HipService/User/UserController.cs [hunk 3]
```diff
@@ -48,11 +48,29 @@
~        }
~
~        [NonAction]
~        public async Task OnAuthConfirmFor(JObject response)
         {
             var authOnConfirmResponse = response.ToObject<AuthOnConfirmResponse>();
```
Comment (ganesanarun) [inline src/In.ProjectEKA.HipService/User/UserController.cs:53]: Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6173534)

**Review by dheeraj-p — APPROVED** [review]

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/Link/Model/Auth.cs  1
- src/In.ProjectEKA.HipService/Link/Model/Patient.cs  1
- src/In.ProjectEKA.HipService/Link/Model/Address.cs  1
- src/In.ProjectEKA.HipService/User/UserController.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6173534&bid=20154212)

File: src/In.ProjectEKA.HipService/User/UserController.cs (hunk 3 above)
Comment (ganesanarun) [inline src/In.ProjectEKA.HipService/User/UserController.cs:53]: Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6173534)

**Pull request merged**